modules/webadmin.cpp

```diff
@@ -1094,6 +1094,8 @@ class CWebAdminMod : public CModule {
                 pNetwork->Clone(*pOldNetwork, false);
                 pUser->DeleteNetwork(pOldNetwork->GetName());
             }
+        } else {
+            pNetwork->SetName(sName);
```

Comment:
This will crash.

Reply:
Ah, no, it uses || there

Reply:
Reversing the condition to `pNetwork && Equals()` would make it less confusing

Reply:
I think I'm following. So like this?

```cpp
if (pNetwork && pNetwork->GetName().Equals(sName)) {
    pNetwork->SetName(sName);
} else {
    // AddNetwork()
}
```

Reply:
Yep, with a comment what SetName is supposed to do there